tests/kafkatest/services/kafka_log4j_appender.py

```diff
@@ -49,10 +49,18 @@ def start_cmd(self, node):
 
         if self.max_messages > 0:
             cmd += " --max-messages %s" % str(self.max_messages)
-        if self.security_protocol == SecurityConfig.SSL:
-            cmd += " --security-protocol SSL"
+        if self.security_protocol != SecurityConfig.PLAINTEXT:
+            cmd += " --security-protocol %s" % str(self.security_protocol)
+        if self.security_protocol == SecurityConfig.SSL or self.security_protocol == SecurityConfig.SASL_SSL:
             cmd += " --ssl-truststore-location %s" % str(SecurityConfig.TRUSTSTORE_PATH)
             cmd += " --ssl-truststore-password %s" % str(SecurityConfig.ssl_stores['ssl.truststore.password'])
+        if self.security_protocol == SecurityConfig.SASL_PLAINTEXT or \
```

Comment:
Not a big deal, but for future reference doing something like `if self.security_protocol in [SecurityConfig.SASL_PLAINTEXT, SecurityConfig.SASL_SSL, SecurityConfig.SASL_MECHANISM_GSSAPI, SecurityConfig.SASL_MECHANISM_PLAIN]` is a more concise way to express conditions like this. In fact, it might be valuable to add those collections to `SecurityConfig` since it would allow for simple tests that look something like `if self.security_protocol in SecurityConfig.SASL_MECHANISMS`.